app/controllers/admin_controller.rb

```diff
@@ -102,4 +104,29 @@ def culled_games
   def about
   end
 
+  def stats_display
+    games = Event.current.top_games()
+    titles = Game.where(culled: false).size
+    active_checkouts = Checkout.where(event: Event.current, closed: false).size
+    checked_out = number_to_percentage(active_checkouts.to_f / titles, precision: 0)
+    titles_played = Checkout.where(event: Event.current).joins(game: [:title]).group(:title).count.count
+    total_checkouts = Checkout.where(event: Event.current).size
+    longest_checkout = Checkout.longest_checkout_game_today(Event.current.utc_offset)
+    hours_played = (Checkout.where(event: Event.current).map {|c| c.return_time ? c.return_time.to_f - c.check_out_time.to_f : DateTime.now.to_f - c.check_out_time.to_f }.reduce(:+) / 3600).to_i
```

Comment:
This should be a multi-line block, not a single-liner. Additionally, some of this code is better suited to living in the model.

In checkout.rb, add a method:

```ruby
def hours_played
  if return_time
    return_time - check_out_time
  else
    DateTime.now - check_out_time
  end
end
```

Then, here:

```ruby
total_hours_played = Checkout.for_current_event.sum(:hours_played)  / 1.hour
```

I believe that *should* work